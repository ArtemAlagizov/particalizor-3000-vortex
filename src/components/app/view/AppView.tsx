import React from "react";
import { IAppView } from "./IAppView";
import ParticleVortexShowcase from "../../particle-vortex-showcase/container/ParticleVortexShowcase";

const AppView: React.FC<IAppView> = ({}) => {
  return <ParticleVortexShowcase imageHeight={100} imageWidth={80} />;
};

export default AppView;
