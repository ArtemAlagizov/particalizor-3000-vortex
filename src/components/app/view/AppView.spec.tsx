import React from "react";
import { shallow } from "enzyme";
import AppView from "./AppView";

describe("Given a AppView", () => {
  describe("when rendering", () => {
    it("should match snapshot", () => {
      const component = shallow(<AppView />);

      expect(component).toMatchSnapshot();
    });
  });
});
