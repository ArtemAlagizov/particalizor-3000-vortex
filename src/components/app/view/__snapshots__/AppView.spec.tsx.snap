// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a AppView when rendering should match snapshot 1`] = `
<BrowserRouter
  basename="/relaxator"
>
  <Route
    exact={true}
    path="/"
  >
    <ParticleVortex />
  </Route>
</BrowserRouter>
`;
